analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -818,6 +838,7 @@ class CompileActionUniqueingType(object):
 
 
 def parse_unique_log(compilation_database,
+                     report_dir,
```

Comment:
please texted the documentation that a `compiler_info.json` is witten out to the reports directory too.
Why do we need to do that here? It has not much connection to log parsing.

Reply:
Not depending on that the log parsing tests do not need any changes,

Reply:
it is documented now. Later on, we may want to factor out compiler auto-detection from parse_unique_log.